content/docs/programs/requisitions.md

```diff
@@ -73,6 +73,55 @@ The list view has gained some additional columns, showing the program related da
 - **Order type**: the name of the order type ( typically, a normal or emergency order )
 - **Period**: the name of the period selected
 
-And the detail view also has program information shown:
+#### Manual Program Requisitions
+
+You can create a manual program requisition for customers who do not use mSupply
+and those who do (for those who do, this should only be done in exceptional).
+
+See the configuration steps above, and also the [Customer
+tag](https://docs.msupply.org.nz/names:adding_and_editing#tags_tab)
+documentation to find out how to set up programs and add tags to customers.
+
+Click on the `New Requisition` button (top right corner).
+
+![New Requisition](/docs/programs/images/add_requisition_button.png)
+
+If you have programs set up, you'll see a different modal to the usual one:
+
+![New Program Requisition](/docs/programs/images/add_program_requisition.png)
+
+Here, you can select the:
+
+- Program
+- Customer
+- Order Type
+- Period
+
+Only when you have selected all of the above, can you click `Create` and
+continue.
```

Comment:
```suggestion
continue.

Use the `General` tab to create a regular Manual Requisition`.
```
and maybe link to manual req docs?

Reply:
This feels very random here.... If they're reading the program requisition docs you assume they want to create a program requisition not a normal one no?